src/config-loader/config-loader.ts

```diff
@@ -282,14 +310,36 @@ export const setup = (app: Express.Application) => {
 				}),
 			);
 			await loadConfig(configObj);
+			await loadAsyncMigrations(configObj);
 		} catch (err) {
 			console.error('Error loading application config', err, err.stack);
 			process.exit(1);
 		}
 	};
 
+	const loadAsyncMigrations = async (data: Config): Promise<any> => {
+		try {
+			await Promise.all(
+				data.models.map(async (model) => {
+					if (
+						model.migrations != null &&
+						Object.keys(model.migrations).includes(migrationCategories.async)
+					) {
+						await sbvrUtils.executeAsyncModelMigration(
+							model as sbvrUtils.ExecutableModel,
+						);
+					}
+				}),
+			);
+			return Promise.resolve('Async migrations executed');
+		} catch (err) {
+			return Promise.reject(`Error loading async migrations: \n ${err}`);
+		}
+	};
+
 	return {
 		loadConfig,
 		loadApplicationConfig,
+		loadAsyncMigrations,
```

Comment:
Why do you export this? Is it used elsewhere apart from above in `loadApplicationConfig`?

Reply:
No its not exported or used externally. Thx.